### ISSUE TITLE
lxdock shell with user defined without home defined tries to set HOME=/root

### ISSUE THREAD

**Issue by shuhaowu** [lxdock/lxdock#116]
From what I can tell, the way we enter a container is something like:

```
lxc exec container-name [--env HOME=/home/set/in/lxdockyml] -- su -m user
```

Is there a reason why we don't just use `su -l user`? According to man:

       -, -l, --login
           Provide an environment similar to what the user would expect had the user logged in directly.
       -m, -p, --preserve-environment
           Preserve the current environment, except for:

           $PATH
               reset according to the /etc/login.defs options ENV_PATH or ENV_SUPATH (see below);

           $IFS
               reset to “<space><tab><newline>”, if it was set.

           If the target user has a restricted shell, this option has no effect (unless su is called by root).

           Note that the default behavior for the environment is the following:

               The $HOME, $SHELL, $USER, $LOGNAME, $PATH, and $IFS environment variables are reset.

               If --login is not used, the environment is copied, except for the variables above.

               If --login is used, the $TERM, $COLORTERM, $DISPLAY, and $XAUTHORITY environment variables are copied if they were set.

               Other environments might be set by PAM modules.

**Comment by shuhaowu** [comment]
For the record, this is what happens if you have just an user entry under shell (no home defined):

```
$ lxdock shell                                                                                                                         
bash: /root/.bashrc: Permission denied
dev@test-lxdock-test-lxdock-23307078:~$ echo $HOME
/root
```

**Comment by robvdl** [comment]
Thank you, not only have you fixed this, but also #66 which was on my todo list.
